### PR TITLE
Adds support for FIPS option

### PR DESCRIPTION
**Pull request by baloo**

### CODE REVIEW

File: src/client.rs [hunk 1]
```diff
@@ -900,6 +919,26 @@ impl Client {
         Ok(())
     }
 
+    /// Put the FIPS global option: when enabled, it disables algorithms that are
+    /// not allowed by FIPS 140. This also enables the forced auditing global option
+    /// and command logging.
+    ///
+    /// Options are `Off`, or `Fix` (i.e. fixed permanently on)
```
Comment (baloo) [inline src/client.rs:926]:
```suggestion
    /// Options are `Off`, or `On` (Can't use `Fix`)
```

**Comment by tony-iqlusion** [comment]
@baloo this seems simple enough. Why is it marked as draft?

**Review by tony-iqlusion — APPROVED** [review]

**Comment by baloo** [comment]
Because I've made a first version of it that I didn't test (documenting support for `Fix` and not just `On`/`Off`) but it ended up that `Fix` always returns an error.

This version is correct, I just forgot to remove the draft status.

**Comment by baloo** [comment]
Because I've made a first version of it that I didn't test (documenting support for `Fix` and not just `On`/`Off`) but it ended up that `Fix` always returns an error.

This version is correct, I just forgot to remove the draft status.

**Pull request merged**